stable/jenkins/templates/config.yaml

```diff
@@ -116,6 +116,10 @@ data:
               <imagePullSecrets/>
 {{- end }}
               <nodeProperties/>
+{{- if .Values.agent.yamlTemplate }}
+              <yaml>
```

Comment:
You can also write it like this to avoid the newline at the beginning within the Jenkins UI:
`<yaml>{{- tpl .Values.agent.yamlTemplate . | html | indent 4 | trim}}</yaml>`

> This will remove the whitespace from the first line.
https://github.com/Masterminds/sprig/issues/54

Reply:
Let me fix that I tried for a little while to get rid of the leading newline but wasn't able to.